plugins/discourse-user-notes/app/controllers/discourse_user_notes/user_notes_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
       user = User.where(id: params[:user_id]).first
       raise Discourse::NotFound if user.blank?
 
+      # TODO: add modifier?
```

Comment:
Why are we adding a todo? Especially on the stable branch?

I feel like we should either action it, or delete the comment. Otherwise it'll sit here forever.